### PR TITLE
Ana/enable insecure signups in development

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Again, very small changes, but I think a significant improvement.

It was getting kind of monotonous to have to enter our secret flag all the time. This way it saves up both time and mental power.

Now we have all the signups options in development, and if we want to try things out in production, we just need to set `produtctionMode` to true by using the `?production=true` flag

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
This bit doesn't seem to be working as expected...?

It always routes to the SelectNetwork component. Or is that expected?

```
    createAccountLink() {
      return this.isMobileApp ||
        this.session.insecureMode
        ? `/select-network/create`
        : `/create`
    }
```

File: src/components/common/CardSignInRequired.vue [hunk 1]
```diff
@@ -5,7 +5,11 @@
 
     <div class="session-list">
       <LiSession
-        v-if="session.insecureMode || (isMobileApp && !accountExists)"
+        v-if="
+          session.insecureMode ||
+            (session.developmentMode && !session.productionMode) ||
```
Comment (faboweb) [inline src/components/common/CardSignInRequired.vue:10]: this looks weird

**Comment by faboweb** [comment]
why not just set insecureMode to true if in development?

**Comment by Bitcoinera** [comment]
> why not just set insecureMode to true if in development?

true 😅 ...

**Comment by Bitcoinera** [comment]
Anyway, I was just thinking, maybe this soon won't be necessary right? Because as soon as we release Kusama, we will need to make these ones available in production too.

Or what is the plan here? Make available only for the Kusama network?

File: src/initializeApp.js [hunk 2]
```diff
@@ -27,6 +27,9 @@ function setOptions(urlParams, store) {
   if (urlParams.insecure === `true`) {
     store.commit(`setInsecureMode`)
   }
+  if (urlParams.production === `true`) {
```
Comment (faboweb) [inline src/initializeApp.js:30]: remove production mode switch?

Comment (Bitcoinera) [reply src/initializeApp.js:30]: But then we cannot test the `/create` endpoint in development. I mean this bit:

image

Comment (faboweb) [reply src/initializeApp.js:30]: In doubt you can just manually change the value in code. This is a very rare case

Comment (Bitcoinera) [reply src/initializeApp.js:30]: OK, then I will remove it

**Pull request merged**